config/locales/en.yml

```diff
@@ -33,6 +33,7 @@ en:
   date:
     formats:
       default: "%-d %B %Y"
+      claim_checking: "%d/%m/%Y"
```

Comment:
Actually, can we use a more generic name here? (such as `date_month_year`). We might want to use it elsewhere.